src/utils/constants/constants.ts

```diff
@@ -41,21 +47,30 @@ export const DPI_SINGLE_INDEX_MODULE =
 export const TEN_POW_18 = BigNumber.from(10).pow(18)
 export const TEN_POW_16 = BigNumber.from(10).pow(16)
 
-export const ALCHEMY_API =
-  'https://eth-mainnet.alchemyapi.io/v2/5j2PCDrDSbB5C6n8pnka21H3NSoUje4j' // + process.env.ALCHEMY_TOKEN;
+export const ALCHEMY_API = {
+  [ChainId.ethereum]:
+    'https://eth-mainnet.alchemyapi.io/v2/5j2PCDrDSbB5C6n8pnka21H3NSoUje4j', // + process.env.ALCHEMY_TOKEN;
+  [ChainId.polygon]:
+    'https://polygon-mainnet.g.alchemy.com/v2/L9jbLT6_-U9rEuvLoMXVvUYoZQx04pEI',
```

Comment:
this is just a test account, let me know if have another one I should add.

Reply:
When this is finalized, let's replace this with a server variable